src/components/print/PrintDirective.js

```diff
@@ -387,8 +379,8 @@ goog.require('ga_urlutils_service');
                   display: [$scope.layout.map.width, $scope.layout.map.height],
                   // scale has to be one of the advertise by the print server
                   scale: $scope.scale.value,
-                  dataOwner: dataOwner,
-                  thirdPartyDataOwner: thirdPartyDataOwner,
+                  dataOwner: allDataOwner,
+                  thirdPartyDataOwner: false,
```

Comment:
can we remove it or it breaks the print ?

Reply:
Yes, me may. I does not break